pom.xml

```diff
@@ -82,6 +82,7 @@
     <module>apollo-client-config-data</module>
     <module>apollo-mockserver</module>
     <module>apollo-openapi</module>
+    <module>apollo-plugin</module>
```

Comment:
About the maven module

* apollo-client-plugin/apollo-client-plugin-log4j2

is better?

Reply:
If you want to add the prefix for the submodule, how about `apollo-plugin/apollo-plugin-log4j2`?

Reply:
It's ok.